test/index.test.js

```diff
@@ -279,6 +280,24 @@ describe('Segment.io', function() {
         analytics.assert(object.context);
         analytics.assert(!object.context.amp);
       });
+
+      it('should add a list of enabled integrations when `addEnabledMetadata` is set', function() {
```

Comment:
I added a test that `send` sends `normalize`d data, but I'm not sure how much sense it actually makes.